internal/service/device.go

```diff
@@ -519,6 +521,50 @@ func (h *ServiceHandler) UpdateServiceSideDeviceStatus(ctx context.Context, devi
 	return common.UpdateServiceSideStatus(ctx, orgId, &device, h.store, h.log)
 }
 
+func (h *ServiceHandler) ResumeDevices(ctx context.Context, request api.DeviceResumeRequest) (int, api.Status) {
+	orgId := getOrgIdFromContext(ctx)
+
+	// Normalize selectors by trimming whitespace and treating empty strings as nil
+	var normalizedLabelSelector *string
+	var normalizedFieldSelector *string
+
+	if request.LabelSelector != nil {
+		trimmed := strings.TrimSpace(*request.LabelSelector)
+		if trimmed != "" {
+			normalizedLabelSelector = &trimmed
+		}
+	}
+
+	if request.FieldSelector != nil {
+		trimmed := strings.TrimSpace(*request.FieldSelector)
+		if trimmed != "" {
+			normalizedFieldSelector = &trimmed
+		}
+	}
+
+	// Validate that at least one selector is provided after normalization
+	if normalizedLabelSelector == nil && normalizedFieldSelector == nil {
+		return 0, api.StatusBadRequest("at least one non-empty selector (labelSelector or fieldSelector) must be provided to avoid accidental resume all")
+	}
```

Comment:
Like I said above, I understand the intention here, but I'm not sure we want to enforce it.
Maybe we should add a "--all" flag in the CLI and enforce it only there?
Please ask Paola (UX).

Reply:
added all flag based on our talk